packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -255,7 +255,7 @@ class ScriptTransformer {
           let transformer: Transformer;
 
           try {
-            transformer = require(transformPath);
+            transformer = interopRequireDefault(require(transformPath)).default;
```

Comment:
@SimenB This incorrectly assumes that a Jest transformer does not use `export const createTransformer` instead of `export default { createTransformer }`.